src/client/base.js

```diff
@@ -113,11 +113,10 @@ spf.dispatch = function(name, opt_detail) {
  */
```

Comment:
Does the @return need to be updated? perf.timing would return a float while Date() returns an int.